### PR TITLE
Lets the kinetic crusher be one-hand carried

### PR DESCRIPTION
**Pull request by kriskog**
## About The Pull Request
Changes the kinetic crusher from a twohand/required weapon to a twohand optionally wielded weapon. Trying to hit someone with it while one-hand carrying will make you drop it and everything else you are holding.

Beware though, codersprites. I would prefer the front and back sprites to be slightly different, the angle on the haft is a bit off. If someones willing to improve on that, let me know.

## Why It's Good For The Game
The crusher is fun, but a bit too unwieldy when not in use. This improves on that.

## Changelog
:cl: Skoglol
add: Kinetic crusher can now be one-hand carried. You'll still need two hand to use it.
/:cl:

### CODE REVIEW

**Comment by leon2550** [comment]
bUt ThE cRuShEr Is SuPpOsEd To Be UsElEsS!!!!!1!!!11!!!one!!!
or at least that's what they told me the last time i tried to buff this thing

File: code/modules/mining/equipment/kinetic_crusher.dm [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 	icon_state = "crusher"
-	item_state = "crusher"
+	item_state = "crusher0"
 	lefthand_file = 'icons/mob/inhands/weapons/hammers_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/hammers_righthand.dmi'
 	name = "proto-kinetic crusher"
 	desc = "An early design of the proto-kinetic accelerator, it is little more than a combination of various mining tools cobbled together, forming a high-tech club. \
 	While it is an effective mining tool, it did little to aid any but the most skilled and/or suicidal miners against local fauna."
-	force = 20 //As much as a bone spear, but this is significantly more annoying to carry around due to requiring the use of both hands at all times
+	force = 0 //You can't hit stuff unless wielded
```
Comment (nemvar) [inline code/modules/mining/equipment/kinetic_crusher.dm]: Something is wrong

File: code/modules/mining/equipment/kinetic_crusher.dm [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 	w_class = WEIGHT_CLASS_BULKY
 	slot_flags = ITEM_SLOT_BACK
-	force_unwielded = 20 //It's never not wielded so these are the same
+	force_unwielded = 0
```
Comment (nemvar) [inline code/modules/mining/equipment/kinetic_crusher.dm]: Hmmm

**Comment by FantasticFwoosh** [comment]
You can throw it and the crusher does full damage & has embed chance, i think that aspect has to be nerfed first before people start to make throwing axes out of them.

**Comment by kriskog** [comment]
The crushers throwforce is 5, I think we'll be fine.

**Comment by MimicFaux** [comment]
Things get pretty hairy out in the wastes when you have to drop-drag your weapon of choice to do healing on the fly like every other miner type. This will be a welcome change.

**Comment by OverlyGeneric** [comment]
how the hell are you suppose to hold something steady in one-hand when its very top-heavy?

i'm confused how this works.

**Comment by kriskog** [comment]
By the haft, just under the head. Like you would a sledgehammer. Theres a sprite for it.

**Comment by ExcessiveUseOfCobblestone** [comment]
Nice, this was incredibly annoying to play inventory tetris when you weren't even using the thing.

**Comment by Dennok** [comment]
> how the hell are you suppose to hold something steady in one-hand when its very top-heavy?
> 
> i'm confused how this works.

I think you just drag it on floor when weld it handle, but you can't raise it up and hit someone by it.

**Comment by adamkad1** [comment]
a bit late but i really wish this was made like l6 saw instead, dropping them damn thing when you try to fire it one handed is annoying

**Pull request merged**